### PR TITLE
separate webpack config

### PR DESCRIPTION
**Pull request by chasssssse**

### CODE REVIEW

**Review by aliceshen227 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,10 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "npm-run-all --sequential build start:backend",
+    "start:backend": "node ./src/server/index.js",
+    "build": "webpack -d --colors --config webpack.prod.js",
+    "dev:frontend": "webpack-dev-server --colors --config webpack.dev.js --port 3000",
+    "dev:backend": "nodemon ./src/server/index.js",
     "test:backend": "mocha",
-    "test:frontend": "jest --colors",
```
Comment (ikarasz) [inline package.json]: why did you remove our test command?

**Review by ikarasz — CHANGES_REQUESTED** [review]
please use constants in the common webpack config instead of vars and also pay attention to the empty line after the require declarations

File: package.json (hunk 1 above)
Comment (chasssssse) [reply package.json]: I commit before you merge so I don't have the test command

**Review by sliu102 — APPROVED** [review]

**Review by ikarasz — APPROVED** [review]
